src/deploy.ts

```diff
@@ -14,6 +14,7 @@ export async function deploy(): Promise<number> {
     PLATFORM_PROJECT_ID: core.getInput('project-id'),
     PLATFORMSH_CLI_TOKEN: core.getInput('cli-token'),
     FORCE_PUSH: core.getInput('force-push'),
+    PARENT_ENVIRONMENT_NAME: core.getInput('parent-environment-name'),
```

Comment:
I think you forgot to update `action.yml`.

Reply:
I did! I've added new input in the `action.yml`. I also fixed the wording of the description of the input, both in the `README.md` and the `action.yml`.